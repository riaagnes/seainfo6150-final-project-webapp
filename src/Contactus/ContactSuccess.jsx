import React from "react";
import styles from "../Contactus/ContactSuccess.module.css";
const ContactSuccess = () => {
  return (
    <div className={styles.container}>
      <h1 className={styles.form_success}>We have received your request!</h1>
      <img
        className={styles.form_img_2}
        src="https://raw.githubusercontent.com/briancodex/react-form-v1/286f4a4603bda257ae001dc57c74d7f30bd4eedb/public/img/img-3.svg"
        alt="success"
      />
    </div>
  );
};

export default ContactSuccess;
