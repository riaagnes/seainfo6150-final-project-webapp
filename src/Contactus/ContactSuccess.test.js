import React from "react";
import ContactSuccess from "./ContactSuccess";

describe("ContactSuccess tests", () => {
  it("renders correctly", () => {
    const { container } = render(<ContactSuccess  />);
    expect(container).toMatchSnapshot();
  });
});