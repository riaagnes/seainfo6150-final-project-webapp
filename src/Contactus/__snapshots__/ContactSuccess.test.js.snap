// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactSuccess tests renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <h1
      class="form_success"
    >
      We have received your request!
    </h1>
    <img
      alt="success"
      class="form_img_2"
      src="https://raw.githubusercontent.com/briancodex/react-form-v1/286f4a4603bda257ae001dc57c74d7f30bd4eedb/public/img/img-3.svg"
    />
  </div>
</div>
`;
